error: assignment discards ‘const’ qualifier from pointer target type

After adding the fos_custom_ttl_backend_response to vcl_response, i get this error:

```
oct. 31 15:05:43 pc-venitron varnishd[14291]: Message from C-compiler:
oct. 31 15:05:43 pc-venitron varnishd[14291]: vgc.c: In function ‘VGC_function_fos_custom_ttl_backend_response’:
oct. 31 15:05:43 pc-venitron varnishd[14291]: vgc.c:3368:17: error: assignment discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]
oct. 31 15:05:43 pc-venitron varnishd[14291]: ttl = VRT_GetHdr(ctx, &hdr);
oct. 31 15:05:43 pc-venitron varnishd[14291]: ^
oct. 31 15:05:43 pc-venitron varnishd[14291]: cc1: all warnings being treated as errors
oct. 31 15:05:43 pc-venitron varnishd[14291]: Running C-compiler failed, exited with 1
```

I followed the doc install but it fails due this error.

I'm using ubuntu 15.10 and varnish-4.1.0 revision 3041728

As in doc said, i had enabled the C inline to use this feature  by setting this config in varnish.service (The new systemd tool in ubuntu 15.10). Here is the config:

```
[Unit]
Description=Varnish Cache, a high-performance HTTP accelerator

[Service]
Type=forking
LimitNOFILE=131072
LimitMEMLOCK=82000
ExecStartPre=/usr/sbin/varnishd -C -f /etc/varnish/default.vcl -p vcc_allow_inline_c=on
ExecStart=/usr/sbin/varnishd -a :80 -T localhost:6082 -f /etc/varnish/default.vcl -S /etc/varnish/secret -s malloc,512m -p vcc_allow_inline_c=on
ExecReload=/usr/share/varnish/reload-vcl

[Install]
WantedBy=multi-user.target
```

Any Help ?

Thank you

--- a/resources/config/varnish-4/fos_custom_ttl.vcl
+++ b/resources/config/varnish-4/fos_custom_ttl.vcl
@@ -26,7 +26,7 @@ sub fos_custom_ttl_backend_response {
          * startup command.
          */
         C{
-            char *ttl;
+            const char *ttl;
             const struct gethdr_s hdr = { HDR_BERESP, "\024X-Reverse-Proxy-TTL:" };
             ttl = VRT_GetHdr(ctx, &hdr);
             VRT_l_beresp_ttl(ctx, atoi(ttl));

